packages/transitive-overlay/src/index.tsx

```diff
@@ -15,24 +15,58 @@ import bbox from "@turf/bbox";
 import { getRouteLayerLayout, patternToRouteFeature } from "./route-layers";
 import { drawArc, getFromToAnchors, itineraryToTransitive } from "./util";
 import routeArrow from "./images/route_arrow.png";
-import one from "./images/01.png";
-import three from "./images/03.png";
-import four from "./images/04.png";
-import five from "./images/05.png";
-import six from "./images/06.png";
-import seven from "./images/07.png";
-import eight from "./images/08.png";
-import nine from "./images/09.png";
-import ten from "./images/10.png";
-import eleven from "./images/11.png";
-import twelve from "./images/12.png";
-import thirteen from "./images/13.png";
-import fourteen from "./images/14.png";
-import fifteen from "./images/15.png";
-import sixteen from "./images/16.png";
-import seventeen from "./images/17.png";
+import capsule1 from "./images/01.png";
```

Comment:
For subsequent PR: The images are being downloaded all, even though only one/a subset is actually used.